src/Compilers/CSharp/Portable/Binder/Binder_Operators.cs

```diff
@@ -2303,6 +2303,7 @@ private void ReportSuppressionIfNeeded(BoundExpression expr, DiagnosticBag diagn
             }
         }
 
+#nullable enable
```

Comment:
Note: I enabled nullable here and in the extension because it caught a bug, namely blindly dereferencing `Type`.